samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -746,6 +761,117 @@ object SamzaContainer extends Logging {
       containerStorageManager = containerStorageManager)
   }
 
+
+  private[container] def getStandbyTasks(jobModel: JobModel, containerModel: ContainerModel,
+    consumerMultiplexer: SystemConsumers, producerMultiplexer: SystemProducers, offsetManager: OffsetManager,
+    timerExecutor: ScheduledExecutorService, reporters: Map[String, MetricsReporter],
+    storageEngineFactories: Map[String, StorageEngineFactory[Object, Object]], loggedStorageBaseDir: File,
+    storeWatchPaths: util.HashSet[Path], jobContext: JobContext, containerContext: ContainerContext,
+    changeLogSystemStreams: Map[String, SystemStream], streamMetadataCache: StreamMetadataCache,
+    systemAdmins: SystemAdmins, config: Config, applicationContainerContextOption: Option[ApplicationContainerContext],
+    applicationTaskContextFactoryOption: Option[ApplicationTaskContextFactory[ApplicationTaskContext]],
+    externalContextOption: Option[ExternalContext])
+  : Map[TaskName, TaskInstance] = {
+
+    containerModel.getTasks.values.asScala.
+      filter(_.getTaskMode == TaskMode.Standby).map(taskModel => {
+      debug("Setting up active task instance: %s" format taskModel)
```

Comment:
Everything under map should be indented by +2. See formatting suggestion earlier.